app/screens/phone-auth-screen/phone-auth.tsx

```diff
@@ -317,6 +347,24 @@ export const WelcomePhoneValidationScreen: ScreenType = ({
             >
               {code}
             </Input>
+            <View style={styles.timerRow}>
+              <Pressable
+                onPress={() => {
+                  if (!loading && timer === 0) {
+                    sendCodeAgain()
+                  }
+                }}
+              >
+                <Text
+                  style={{
+                    color: timer === 0 ? color.palette.black : color.palette.midGrey,
+                  }}
+                >
+                  {translate("WelcomePhoneValidationScreen.sendAgain")}
+                </Text>
+              </Pressable>
+              <Timer onTimerFinish={() => setTimer(0)} count={timer} />
```

Comment:
I think hiding the timer might look good once it reaches 0.

Reply:
I changed it to show timer when it's more than 0 and also show `Send Again` button when timer is 0.